core/src/tasks/profile/enqueueExports.ts

```diff
@@ -0,0 +1,49 @@
+import { Op } from "sequelize";
+import { Profile } from "../../models/Profile";
+import { CLSTask } from "../../classes/tasks/clsTask";
+import { plugin } from "../../modules/plugin";
+import { Import } from "../../models/Import";
+import { CLS } from "../../modules/cls";
+
+export class ProfilesEnqueueExports extends CLSTask {
+  constructor() {
+    super();
+    this.name = "profiles:enqueueExports";
+    this.description =
+      "If a profile is ready and has an import that has not exported, enqueue profile:export for them";
+    this.frequency = 1000 * 10;
+    this.queue = "profiles";
+    this.inputs = {};
+  }
+
+  async runWithinTransaction() {
+    const limit = parseInt(
+      (await plugin.readSetting("core", "runs-profile-batch-size")).value
```

Comment:
👍 I think this is the right setting to use.
The `exports-profile-batch-size` setting will be used later on to /send/ the exports, but here we are /creating/ the exports.